C/CoinUtils/build_tarballs.jl

```diff
@@ -0,0 +1,82 @@
+  for sym in ${syms[@]}
+  do
+    CPPLAGS+=("-D${sym}=${sym}_64")
+  done
+fi
+
+configure_args=()
+if [[ ${target} == *darwin* || ${target} == arm-* || ${target} == *freebsd* ]]; then
+  configure_args+=(--host=${target})
+fi
+
+CPPFLAGS="${CPPFLAGS[@]}" ./configure --prefix=${prefix} --with-blas-lib="${OPENBLAS[@]}" ${configure_args[@]}
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = expand_cxxstring_abis(supported_platforms())
+# platforms = [
+#  Linux(:i686, libc=:glibc)
+#  Linux(:x86_64, libc=:glibc)
+#  # Linux(:aarch64, libc=:glibc)  # fails
+#  Linux(:armv7l, libc=:glibc, call_abi=:eabihf)
+#  # Linux(:powerpc64le, libc=:glibc) # fails
+#  Linux(:i686, libc=:musl)
+#  Linux(:x86_64, libc=:musl)
+#  # Linux(:aarch64, libc=:musl)  # fails
+#  # Linux(:armv7l, libc=:musl, call_abi=:eabihf)  # fails
+#  MacOS(:x86_64)
+#  FreeBSD(:x86_64)
+#  # Windows(:i686)  # fails
+#  # Windows(:x86_64)  # fails
+# ]
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libCoinUtils", :libCoinUtils)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = Dependency[
+    Dependency("OpenBLAS_jll"),
+]
+
+# Build the tarballs, and possibly a `build.jl` as well.
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies)
```

Comment:
Given that GLPK and GMP are `using preferred_gcc_version=v"6"`, I wonder if we should do that throughout all the JuMP libraries, because they all tend to link to one another.